src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -576,6 +576,7 @@ private function addAssetsSection(ArrayNodeDefinition $rootNode)
                         ->scalarNode('version')->defaultNull()->end()
                         ->scalarNode('version_format')->defaultValue('%%s?%%s')->end()
                         ->scalarNode('json_manifest_path')->defaultNull()->end()
+                        ->scalarNode('json_manifest_url')->defaultNull()->end()
```

Comment:
can't we allow passing a URL to `json_manifest_path` instead?

Reply:
But in this case rename the option (+deprecation) to json_manifest too?

Reply:
the less deprecation the better to me...

Reply:
Same opinion on this, but the DX would be worse. We can fix this issue in the docs, sure

Reply:
Sure, http(s) urls can be detected like this:
https://github.com/symfony/symfony/pull/35762/files#diff-0e793081ceb720201745c982a568903fR1049
`'http' === substr(parse_url($jsonManifestPath, \PHP_URL_SCHEME), 0, 4)`